teams.py

```diff
@@ -25,9 +25,9 @@ def hero_generator(self):
         generates particular hero
         :return:
         '''
-        types_of_characters = [character.Sorceress, character.Warrior, character.Support]
-        chosen_char = types_of_characters[random.randint(0, 2)]
-        if chosen_char is character.Sorceress:
+        types_of_characters = [characters.Sorceress, characters.Warrior, characters.Support, characters.Voodoo]
+        chosen_char = types_of_characters[random.randint(0, len(types_of_characters) - 1)]
+        if chosen_char is characters.Sorceress or chosen_char is characters.Voodoo:
```

Comment:
możesz zrobić tak:
if chosen_char in [characters.Sorceress, characters.Voodoo]

Reply:
ok